src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -482,8 +504,101 @@ const Authenticate = (() => {
         }
     };
 
+    const initializeTab = () => {
+        TabSelector.onLoad();
+    };
+
+    const getAuthenticationStatus = () => new Promise((resolve) => {
+        const mock_response = {
+            needs_verification: ['identity', 'document'],
+            identity          : {
+                status     : 'none',
+                expiry_date: 0,
+            },
+            document: {
+                status     : 'none',
+                expiry_date: 0,
+            },
+        };
+        setTimeout(() => {
+            resolve(mock_response);
+        }, 300);
+    });
+
+    const AuthenticationInit = async () => {
+        // TODO: call the authentication API
+        const authentication_status = await getAuthenticationStatus();
+        if (!authentication_status && authentication_status.error) return;
```

Comment:
this can't both be true, either we don't have the status or we have it and there's an error in it. also we should handle the error